curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-35-circular-primes.english.md

```diff
@@ -106,7 +106,7 @@ function circularPrimes(n) {
           continue;
         }
         else if (!primes[x]) {
-          // If the rotated value is 0 then its not a ciruclar prime, break the loop
+          // If the rotated value is 0 then it isn't a circular prime, break the loop
```

Comment:
`its` and `it's` are not the same. As a general rule, it's a lot safer to write `isn't` when `is` is the verb, as you won't confuse them.